### PR TITLE
Should not proceed if endpoint is none for heketi service

### PR DESCRIPTION
**Pull request by MohamedAshiqrh**
Signed-off-by: Mohamed Ashiq Liyazudeen <mliyazud@redhat.com>

---
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/208)

### CODE REVIEW

**Comment by jarrpa** [comment]
I don't see the need for this. If the endpoint is none the `describe svc` call would fail and the service would still be set to the empty string. Am I missing something?

**Comment by MohamedAshiqrh** [comment]
```
Waiting for heketi pod to start ... OK
```
here Got the route name:
```
service endpoint heketi-aplo.cloudapps.myaplo.com
```
Described route which says endpoint none, Which updates in some delay. But we try to curl on it and fail.
```
Name:                   heketi
Namespace:              aplo
Created:                3 seconds ago
Labels:                 glusterfs=heketi-route
                        template=heketi
Annotations:            openshift.io/host.generated=true
Requested Host:         heketi-aplo.cloudapps.myaplo.com
                          exposed on router aplo-router 4 seconds ago
Path:                   <none>
TLS Termination:        <none>
Insecure Policy:        <none>
Endpoint Port:          <all endpoint ports>
 
Service:        heketi
Weight:         100 (100%)
Endpoints:      <none>
Failed to communicate with heketi service.
Please verify that a router has been properly configured.
```

**Comment by MohamedAshiqrh** [comment]
@jarrpa cool done 👍

File: deploy/gk-deploy [hunk 1]
```diff
@@ -527,12 +526,12 @@ check_pods "glusterfs=heketi-pod"
 output "OK"
 
 heketi_service=""
+endpoint=""
```
Comment (jarrpa) [inline deploy/gk-deploy]: Not needed. ;)

**Comment by humblec** [comment]
Thanks @MohamedAshiqrh ! LGTM.

**Comment by jarrpa** [comment]
@humblec And you did it twice, no less!! See my last comment on #205.

**Pull request merged**